src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -273,6 +274,19 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
             return;
         }
 
+        let signupQualifier;
+        if (currentUrl.includes('signupQualifier')) {
+            signupQualifier = new URL(currentUrl).searchParams.get('signupQualifier');
+        }
+        if (signupQualifier) {
```

Comment:
If tou include this in the above if scope we do not need this if condition check, right?

Reply:
Right!